eng/Baseline.Designer.props

```diff
@@ -374,15 +374,15 @@
   </PropertyGroup>
   <ItemGroup Condition=" '$(PackageId)' == 'Microsoft.AspNetCore.DataProtection.StackExchangeRedis' AND '$(TargetFramework)' == 'net462' ">
     <BaselinePackageReference Include="Microsoft.AspNetCore.DataProtection" Version="[8.0.0, )" />
-    <BaselinePackageReference Include="StackExchange.Redis" Version="[2.6.122, )" />
+    <BaselinePackageReference Include="StackExchange.Redis" Version="[2.7.23, )" />
```

Comment:
Was this updated by a repo-wide find/replace? If so please revert, this file should only be updated after a release

Reply:
Yes, my bad; ctrl-f misunderstanding - will revert